CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
Note that jsoncpp is shipped with VTK as well. In the Slicer core we should rather depend on VTK only (Slicer may be built using CLI support disabled), unless there is a strong reason against it.
Is the jsoncpp version in VTK will be synchronized with jsconcpp in SlicerExecutionModel?